File: Programmers/Level2/예상 대진표/예상 대진표.java
class Solution{
    public int solution(int n, int a, int b){
        int answer = 1;
        
        if(a>b){
            int tmp=a;
            a=b;
            b=tmp;
        }
        
        while(true){ 
            if(b-a==1 && b % 2 == 0){
                break;
            }
            if(a%2!=0){
            a=a+1;
            }
            if(b%2!=0){
            b=b+1;
            }
            a=a/2;
            b=b/2;
            answer++;
        }
        
        return answer;
    }
}

File: Programmers/Level2/예상 대진표/예상 대진표.py
def solution(n,a,b):
    answer = 1

    while True:
        if a%2 !=0 and b%2 ==0 and b-a == 1 or a%2 ==0 and b%2 !=0 and a-b ==1:
            return answer
        
        if a%2 !=0:
            a += 1
        if b%2 !=0:
            b += 1
        b /= 2
        a /= 2
        answer += 1  
